docs/source/how_tos/index.rst

```diff
@@ -16,7 +16,8 @@ This guide lists the steps to perform common dsgrid operations.
    set_up_standalone_registry
    run_dsgrid_on_kestrel
    spark_cluster_on_kestrel
+   visualize_data_with_tableau
 .. create_project_base_dimensions
    create_project_supplemental_dimensions
    create_derived_dataset
-   query_project
\ No newline at end of file
+   query_project
```

Comment:
I think I deleted this line because it didn't have any content (yet). Has that changed?

Reply:
It is still commented out. My editor adds a newline character to any file that does not end with one (which is proper UNIX behavior). I think that's why it shows as changed.